apps/provider-console/src/components/deployments/DeploymentDetails.tsx

```diff
@@ -0,0 +1,186 @@
+        const latestBlocks = await consoleClient.get(`/v1/blocks`);
+        const latestBlock = latestBlocks[0].height;
+        const totalAmtSpent = findTotalAmountSpentOnLeases(response.leases, latestBlock, true);
+        const totalCost = totalDeploymentCost(response.leases);
+        const timeLeft = totalDeploymentTimeLeft(response.createdHeight, response.totalMonthlyCostUDenom, latestBlock, response.balance, response.closedHeight);
+        setDeploymentDetails({ ...response, amountSpent: totalAmtSpent, costPerMonth: totalCost, timeLeft: timeLeft });
+        setError(null);
+      } catch (err) {
+        setError("Failed to fetch deployment details");
+        console.error("Error fetching deployment details:", err);
+      } finally {
+        setLoading(false);
+      }
+    };
+
+    fetchDeploymentDetails();
+  }, [dseq, owner]);
+
+  if (loading) {
+    return <div>Loading...</div>;
+  }
+
+  if (error) {
+    return <div>Error: {error}</div>;
+  }
+
+  if (!deploymentDetails) {
+    return <div>No deployment details found</div>;
+  }
+
+  return (
+    <div className="space-y-6">
+      <div className="flex items-center space-x-4">
+        <span className={`rounded-full px-3 py-1 text-sm ${deploymentDetails.status === "active" ? "bg-green-100 text-green-800" : "bg-red-100 text-red-800"}`}>
+          {deploymentDetails.status}
+        </span>
+        <span className="text-sm font-medium">DSEQ: {dseq}</span>
+      </div>
+
+      <div className="grid grid-cols-3 gap-6">
+        <Card>
+          <CardContent className="p-4">
+            <h3 className="mb-2 text-sm text-gray-500">Owner</h3>
+            <p className="truncate font-semibold">{owner}</p>
+          </CardContent>
+        </Card>
+        <Card>
+          <CardContent className="p-4">
+            <h3 className="mb-2 text-sm text-gray-500">Created</h3>
+            <p className="font-semibold">{new Date(deploymentDetails.createdDate).toLocaleDateString()}</p>
+            <p className="text-sm text-gray-500">{new Date(deploymentDetails.createdDate).toLocaleTimeString()}</p>
+          </CardContent>
+        </Card>
+        {(deploymentDetails.closedHeight || deploymentDetails.status === "active") && (
+          <Card>
+            <CardContent className="p-4">
+              <h3 className="mb-2 text-sm text-gray-500">{deploymentDetails.closedHeight ? "Runtime" : "Deployment expires in"}</h3>
+              <p className="font-semibold">{getPrettyTimeFromSeconds(deploymentDetails.timeLeft)}</p>
+            </CardContent>
+          </Card>
+        )}
+      </div>
+      <h2 className="text-lg font-semibold">Finances</h2>
+      <div className="grid grid-cols-4 gap-6">
+        <Card>
+          <CardContent className="p-4">
+            <h3 className="mb-2 text-sm text-gray-500">Cost</h3>
+            <p className="font-semibold">{uaktToAKT(deploymentDetails.totalMonthlyCostUDenom).toFixed(3)} AKT</p>
```

Comment:
There's a `precision` param in `uaktToAKT`

Reply:
Good point, I missed it